doc/HOW-TO/docker-build.rst

```diff
@@ -0,0 +1,29 @@
+Build a Qserv container image
+############################################################
+
+.. note::
+
+   This procedure is launched for each new commit inside `Travis-CI <https://travis-ci.org/github/lsst/qserv>`_.
+   It possible possible to retrieve the tag of the produced Qserv image on top of build log,
+   the log line start with `Building and testing qserv/qserv:<tag>`.
+
+Choose a Qserv dependencies image
+**********************************************************************
```

Comment:
The is a second-level heading, so it needs to use `=` following our style guide. Also, the underline needs to match the length of the text.

For more information, see the [RestructuredText Style Guide on headings](https://developer.lsst.io/restructuredtext/style.html#sections).

Reply:
:+1: